workspace/api/openfdd_bridge/playground_exec.py

```
@@ -0,0 +1,85 @@
+        py_path = env.get("PYTHONPATH", "")
+        env["PYTHONPATH"] = os.pathsep.join(p for p in (str(api_root), py_path) if p)
+        cmd = [
+            sys.executable,
+            "-m",
+            "openfdd_bridge.playground_worker",
+            str(job_path),
+            str(result_path),
+        ]
+        grace = min(10.0, max(2.0, timeout_s * 0.25))
+        try:
+            proc = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)